### ISSUE TITLE
Image fetching broken in Android Pie (API 28)

### ISSUE THREAD

**Issue by SpectralFergus** [SpectralFergus/tarot#29]
After downloading the latest update on my phone (Android Pie, API 28), card images fail to display. Assuming this new version is the problem, as it still runs fine on my emulator (Android Oreo).

**Running an emulator with Android Pie replicates this error. Cards are blank**

Grabbing images from sacred-texts.com takes place in an async task in Adapter.java. This task makes a call to Drawable.createFromStream() which takes in a URL and returns a drawable. After some digging, it appears that Android Pie comes with a new image API. It introduces the ImageDecoder class, which can perform the same task. I believe the issue lies within this image handling here.

**Comment by mohammaduvez** [comment]
No, actually the problem is the URL used `http://www.sacred-texts.com` is `http`.

In Android P [Network TLS enabled by default](https://developer.android.com/about/versions/pie/android-9.0-changes-28#framework-security-changes).

First of all use `https` or set `android:cleartextTrafficPermitted="true"` in manifest.

Using `https` with `URL#getContent()` will not work unless SSL Certificates are handled prperly, insted I recommend to use image loading library.
